lib/starting_page.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
```

Comment:
Is this library required?

Reply:
It must be added through Android Studio code completion, I'll check it's requirement.